mmv1/products/datastream/ConnectionProfile.yaml

```diff
@@ -169,9 +173,12 @@ properties:
         type: String
         description: |
           Password for the Oracle connection.
-        required: true
         sensitive: true
         custom_flatten: 'templates/terraform/custom_flatten/datastream_connection_profile_oracle_profile_password.go.tmpl'
+      - name: 'secret_manager_stored_password'
```

Comment:
Whoops, missed this earlier. The names should be camel cased ie. `secretManagerStoredPassword`

Reply:
Fixed it, thanks!